README.md

```diff
@@ -9,12 +9,32 @@ This example connector:
 
 
 ## Setup
-Check out our [Developer Hub connector guide](https://developers.rossum.ai/docs/your-first-connector) to set up
-and run the connector for the first time.
+You can find more information about webhooks at our [Developer Hub](https://developers.rossum.ai/docs/how-to-use-webhooks).
 
-After setting up the connector, create `CONNECTOR_SECRET_KEY` environment variable in the environment
-where this connector will be running. Its value should be the same as the authentication
-token you created following the connector guide.
+To set the webhook up:
+  * `sudo apt install python3-pip` and `pip3 install -r requirements.txt`
 
-To use this connector for production, use it via HTTPS and enforce the authentication token.
-Customize schema.json to taste before running the connector.
+You can use [elisctl](https://github.com/rossumai/elisctl) tool to configure a Rossum queue to use the webhook.
+
+Create the webhook first:
+
+```
+	 elisctl webhook create "Python Sample Webhook" --active true -e annotation_content.initialize -e annotation_content.user_update --config-url https://yourremoteurl.com/vendor_matching -q QUEUE_ID --config-secret PUT_YOUR_SECRET_HERE
```

Comment:
Maybe add short explanation for PUT_YOUR_SECRET_HERE (similar to SCHEMA_ID and example_schema.json below)?